pkgs/development/python-modules/ormsgpack/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cargo,
+  rustPlatform,
+  rustc,
+}:
+
+buildPythonPackage rec {
+  pname = "ormsgpack";
+  version = "1.9.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aviramha";
+    repo = "ormsgpack";
+    rev = version;
+    hash = "sha256-lFKHXTYtYEjtu+nXemQnB0yjkFD69gr0n7XfJ1Hy3J0=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
```

Comment:
please don't. Please use cargo fetch deps and use the lock in the repo.

Reply:
Thank you. That's what I get for using nix-init. :(

Fixed.